boot_scripts/boot_notebook.bash

```diff
@@ -49,7 +49,8 @@ cat > .jupyter_config.json <<EOF
         "token": "${NOTEBOOK_TOKEN}"
     },
     "FileContentsManager": {
-        "preferred_dir": "${NOTEBOOK_BASE_DIR}/workspace/"
+        "preferred_dir": "${NOTEBOOK_BASE_DIR}/workspace/",
+        "delete_to_trash": false
```

Comment:
Thanks a lot for taking a look. I had realized only later that you were assigned as a reviewer, otherwise I'd add more info before! 😊 Now it should look better.